armi/cases/case.py

```diff
@@ -425,6 +426,33 @@ def _endCoverage(cov=None):
             combinedCoverage.html_report()
             combinedCoverage.xml_report()
 
+    @staticmethod
+    def _getCoverageRcFile(makeCopy=False):
+        """Helper to provide the coverage configuration file according to the OS.
+
+        Parameters
+        ----------
+        makeCopy : bool (optional)
+            Whether or not to copy the coverage config file to an alternate file path
+
+        Returns
+        -------
+        covFile : str
+            path of coveragerc file
+        """
+        covRcDir = os.path.abspath(os.path.join(context.ROOT, ".."))
```

Comment:
Any more robust suggestion here?

Reply:
I think you could also reuse the `PROJECT_ROOT` in the test below.

Reply:
I like this. on it.